crates/rover-client/src/operations/graph/check_workflow/types.rs

```diff
@@ -45,3 +47,40 @@ impl From<WorkflowStatus> for ChangeSeverity {
         }
     }
 }
+
+pub(crate) type WorkflowTaskStatus = graph_check_workflow_query::CheckWorkflowTaskStatus;
+impl From<WorkflowTaskStatus> for ChangeSeverity {
+    fn from(status: WorkflowTaskStatus) -> Self {
+        match status {
+            WorkflowTaskStatus::PASSED => ChangeSeverity::PASS,
+            WorkflowTaskStatus::FAILED => ChangeSeverity::FAIL,
+            WorkflowTaskStatus::PENDING => ChangeSeverity::FAIL,
+            WorkflowTaskStatus::BLOCKED => ChangeSeverity::FAIL,
+            WorkflowTaskStatus::Other(_) => ChangeSeverity::FAIL,
```

Comment:
is this right? should we be adding other change severities? what do these other cases indicate?

Reply:
Good catch. We can remove this and simplify it since it was only used for `OperationCheckResponse.result` which we can switch to just using `OperationCheckResponse.task_status` which is already wired through.

This should also take care of removing extra "result: PASS" in the json output.